include/cantera/PureFluid.h

```diff
@@ -6,17 +6,25 @@
 #ifndef CXX_PUREFLUID
 #define CXX_PUREFLUID
 
+#pragma message("warning: PureFluid.h is deprecated and will be removed after Cantera 2.5.0.")
+
 #include "thermo/PureFluidPhase.h"
 #include "kinetics.h"
 
 namespace Cantera
 {
 
 //! Wrapper for PureFluidPhase with constructor from file
+/*!
+ * @deprecated To be removed after Cantera 2.5.0. Replaceable with PureFluidPhase::initThermoFile
+ */
 class PureFluid : public PureFluidPhase
 {
 public:
-    PureFluid() : m_ok(false), m_r(0) {}
+    PureFluid() : m_ok(false), m_r(0) {
+        warn_deprecated("class PureFluid", "To be removed after Cantera 2.5.0. "
+        "Replaceable with PureFluidPhase::initThermoFile.");
```

Comment:
The default constructor is replaceable with just `PureFluidPhase`. The constructor from input file name is the one that is replaceable by using the default constructor then calling `PureFluidPhase::initThermoFile`.

Also, I would indent the the continuation line by an additional level.